ethdb/batch.go

```diff
@@ -32,11 +32,11 @@ type Batch interface {
 	// Write flushes any accumulated data to disk.
 	Write() error
 
-	// Reset resets the batch for reuse
+	// Reset resets the batch for reuse.
 	Reset()
 
-	// Replay replays the batch into another batch
-	Replay(logger DbEventLogger) error
+	// Replay replays the batch contents.
+	Replay(replayer Replayer) error
```

Comment:
meh... the thing that does the Replay should be a `Replayer`, and the thing which receives them... `Replayee` is a bit awkward...

Reply:
Fixed, force pushed.